mod/05_standard/set/type/uri.rb

```diff
@@ -0,0 +1,11 @@
+format do
+  include Phrase::Format
+
+  view :core do |args|
+    build_link _render_raw(args), render_title(args) || card.name
```

Comment:
interestingly, there is no :title view in the base format class, so it looks like this would break there.  I suppose we should add that view (just make it do the name).

But I think you're right that in HtmlFormat the :title view should not return nil.